### PR TITLE
cmd/tsuru: app-remove should not guess the name of the app #1106

### PR DESCRIPTION
**Pull request by lucasweiblen**
https://github.com/tsuru/tsuru/issues/1106

@fsouza Could you please review this PR? If I understood you correctly that's it, right?

### CODE REVIEW

**Comment by andrewsmedina** [comment]
@lucasweiblen your PR will not fix the problem. To fix it you should remove the lines 119 [0], 138 [1]  that makes the app-remove try to guess the app name and use flagset [2].

[0] - https://github.com/lucasweiblen/tsuru-client/blob/18b43bc5fba069ef240948850b68a683a3a7c00c/tsuru/apps.go#L119
[1] - https://github.com/lucasweiblen/tsuru-client/blob/18b43bc5fba069ef240948850b68a683a3a7c00c/tsuru/apps.go#L138
[2] - https://github.com/tsuru/tsuru/blob/cd4fbd8dacf32411396f1348530c2f32c73cd94c/cmd/guess.go#L114

**Comment by lucasweiblen** [comment]
hey @andrewsmedina , thanks for your feedback :)
just these changes that you talked would be enough to fix the problem? I talked yesterday with Francisco about this issue and the pointed me in this direction of keeping the flags and just testing for an empty string.. maybe I understood something wrong.. I will take a look again tomorrow but we can clarify this on IRC next week.

**Comment by fsouza** [comment]
@lucasweiblen you need to remove the call to c.Guess() and retrieve appName directly from the flag:

```
appName := c.Flags().Lookup("app").Value.String()
```

With this change, the pull request will look good.

Thanks for contributing!

File: tsuru/apps.go [hunk 1]
```diff
@@ -135,9 +135,9 @@ remove it (you are able to remove any app that you see in [[tsuru app-list]]).`,
 }
 
 func (c *appRemove) Run(context *cmd.Context, client *cmd.Client) error {
-	appName, err := c.Guess()
-	if err != nil {
-		return err
+	appName := c.Flags().Lookup("app").Value.String()
+	if appName == "" {
+		return fmt.Errorf("Please specify which app you want to remove.")
```
Comment (fsouza) [inline tsuru/apps.go]: We can improve the error message with something like ``please use the -a/--app flag to specify which app you want to remove``.

Thank you.

Comment (andrewsmedina) [inline tsuru/apps.go]: When you don't want to format the error message you should use `errors.New` instead `fmt.Errorf`.

**Comment by lucasweiblen** [comment]
Done! Thanks for the tips

**Comment by fsouza** [comment]
Thanks!

**Comment by andrewsmedina** [comment]
@lucasweiblen thank you!

**Pull request merged**